src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -515,67 +515,79 @@ Iteration <number>:
 
 ## How to build
 ### Prerequisites
-1. Clone `npu-plugin` repository
-2. Build OpenCV G-API with OpenVINO/ONNXRT support
-#### Build OpenCV G-API with OpenVINO/ONNXRT support
-1. Clone OpenCV repo:
+1. Build OpenCV G-API with OpenVINO/ONNXRT support
+- Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
+    ```
+    mkdir "build" && cd "build"
+    cmake ../ -DCMAKE_BUILD_TYPE=Release     \
+              -DENABLE_PLUGINS_XML=ON        \
+              -DCMAKE_INSTALL_PREFIX=install \
+              -DENABLE_DEBUG_CAPS=ON         \
+              -DENABLE_NPU_DEBUG_CAPS=ON ..
+
+    cmake --build . --config Release --target install --parallel
+    ```
+- Init OpenVINO enviroment
+    ```
+     source "<path-to-openvino-install-dir>/setupvars.sh"
```

Comment:
I think  we could also point setupvars.bat call here for Windows for Windows users:
`<path-to-openvino-install-dir>/setupvars.bat`

Reply:
As I understood we only maintain the sh commands. If we need to specify commands for windows we need to decide what shell we will use (cmd or powershell).